Casperfeng/intoit-frontend2.0
# Config ga

1. Moved routing to seperate folder
2. Added GA-tokens for prod and dev dashboards
3. Implemented historycomponent that tracks routing changes

**To test locally:**
1. Sign in to Google Analytics using the intoit mail
2. Select the "intoit Dev" analytics
3. Run Intoit locally and navigate around the page
4. The live tracker should update
